pom.xml

```diff
@@ -26,8 +26,10 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>6.1.2</cdap.version>
+    <cdap.version>6.8.0-SNAPSHOT</cdap.version>
```

Comment:
you shouldn't need to bump cdap.version as we aren't adding a dependency on CDAP APIs that have been added in 6.8.0-SNAPSHOT